src/api/config/environments/test.rb

```diff
@@ -60,7 +60,9 @@
   end
 
   # TODO: This shouldn't be needed when we switch to RSpec completely
-  config.action_dispatch.rescue_responses.merge!('ActionController::InvalidAuthenticityToken' => 950 )
+  config.action_dispatch.rescue_responses['ActionController::InvalidAuthenticityToken'] = 950
+
+  config.active_job.queue_adapter = :test
```

Comment:
In order to use rspec to test that jobs get queued. See `spec/models/backend_package_spec.rb`